.gitlab-ci.yml

```diff
@@ -112,6 +112,136 @@ build:mender-client-acceptance-testing:
     paths:
       - checksums
 
+.template:build:mender-dist-packages-image:
+  stage: build
+  needs: []
+  image: docker:git
+  tags:
+    - mender-qa-worker-generic
```

Comment:
q: this is what we want, or can we go with weaker machine?

Reply:
We can probably use a weaker machine, yes. I'll compare to the rest of the Docker builders here and align :+1: